cylc/flow/cfgspec/globalcfg.py

```diff
@@ -1487,22 +1498,39 @@ def load(self):
             # Use default locations.
             for conf_type, conf_dir in self.conf_dir_hierarchy:
                 fname = os.path.join(conf_dir, self.CONF_BASENAME)
-                try:
-                    self._load(fname, conf_type)
-                except ParsecError:
-                    LOG.error(f'bad {conf_type} {fname}')
-                    raise
+                self._load(fname, conf_type)
 
         # Expand platforms needs to be performed first because it
         # manipulates the sparse config.
         self._expand_platforms()
+
+        # Flesh out with defaults
+        self.expand()
```

Comment:
This doesn't change the code actually, just makes it clearer - `self._set_default_editors()` was doing the expansion all this time